apps/snitch_core/priv/repo/migrations/20190115093617_add_upn_field_to_products.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Snitch.Repo.Migrations.AddUpnFieldToProducts do
+  use Ecto.Migration
+
+  def change do
+    alter table("snitch_products") do
+      add(:upn, :string)
+    end
+
+    drop unique_index("snitch_products", [:slug])
```

Comment:
@jyotigautam you are dropping the slug here. But, IMO I think this should be done where we remove the product slug to fetch the product. Either make the change here for product search with `UPN`. or remove this drop index here.